cime/src/drivers/mct/shr/seq_flds_mod.F90

```diff
@@ -2292,6 +2292,7 @@ subroutine seq_flds_set(nmlfile, ID, infodata)
     call seq_flds_add(g2x_states_to_lnd,trim(name))
     call seq_flds_add(x2l_states,trim(name))
     call seq_flds_add(x2l_states_from_glc,trim(name))
+    call seq_flds_add(x2o_states,trim(name))
```

Comment:
This should be enclosed in a conditional, `if (trim(cime_model) == 'e3sm') then`

Reply:
Fixed in 59a26f5ec4f1598113a3e68d7f7498c8fa108005